Get-NetView.psm1

```diff
@@ -99,6 +99,21 @@ $ExecFunctions = {
         return $status, $duration.TotalMilliseconds, $commandOut
     } # TestCommand()
 
+    function CreateZip {
```

Comment:
Move CreateZip function to ExecFunctions block to allow for CreateZip to be called within Powershell instances spawned by Start-Thread.